Detectors/ITSMFT/MFT/simulation/include/MFTSimulation/Detector.h

```diff
@@ -49,6 +49,11 @@ namespace mft
 class Detector : public o2::base::DetImpl<Detector>
 {
  public:
+  /// Name : Detector Name
+  /// Active: kTRUE for active detectors (ProcessHits() will be called)
+  ///         kFALSE for inactive detectors
+  Detector(Bool_t active);
```

Comment:
Additional ctor since we would like to be able to disable hit creation. All other detectors had this kind of ctor already